sdk/package.json

```
@@ -81,8 +81,9 @@
   "scripts": {
     "clean": "rm -rf ./dist",
     "copy-files": "cp -r ./proto ./dist && cp index.d.ts ./dist",
-    "compile": "tsc && npm run copy-files",
-    "prepack": "npm run compile",
+    "compile": "tsc --allowJs false --checkJs false && tsc --declaration false --composite false && npm run copy-files",
+    "verify-types": "tsc dist/*.ts dist/src/*.ts --noEmit",
```

Wouldn't be enough to have only this:
```suggestion
    "verify-types": "tsc dist/*.ts --noEmit",
```
?

I think it covers the `dist/src/*.ts` as well. 🤔

---

Would need to be `dist/**/*.ts` if that's supported. I'm removing the generated `*.d.ts` currently anyway, as suggested with avoiding the mixed approach.